job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy

```diff
@@ -41,7 +41,7 @@ class Constants {
     static final String MAIN_BRANCH_PROD_VERSION = '7.14.0'
     static final String KOGITO_MAIN_BRANCH_PROD_VERSION = '2.0.0'
     static final String RHBOP_MAIN_BRANCH_PROD_VERSION = '8.30.0'
-    static final String NEXT_PROD_VERSION = '7.13.3'
+    static final String NEXT_PROD_VERSION = '7.13.4'
     static final String CURRENT_PROD_VERSION = '7.12.2'
```

Comment:
This is also confusing value, is it for the minimal version for upgrade?

Reply:
I would say they are old properties, I checked in the repo and I did not find any reference to it (neither to `BAMOE_CURRENT_PROD_VERSION`)

Reply:
@lampajr your PR Looks good related to the next prod version of RHPAM.

@davidesalerno @spolti related to `CURRENT_PROD_VERSION` and `BAMOE_CURRENT_PROD_VERSION` properties, are they used somewhere in your scripts? I also could not find any reference to those variables in the repository.

Reply:
additional info:
`BAMOE_CURRENT_PROD_VERSION` was added in https://github.com/kiegroup/kie-ci/pull/1425/files#diff-9d55ce9075a526a3684617ae54e1f15ee81673ba5c89794b916086a8ffacc4d9R47

`CURRENT_PROD_VERSION` already existed and had old value 7.12.0 and was updated to `7.12.21 in https://github.com/kiegroup/kie-ci/pull/1395/files#diff-9d55ce9075a526a3684617ae54e1f15ee81673ba5c89794b916086a8ffacc4d9R45

If they are really not used I guess we can remove both.

Reply:
@spolti @davidesalerno could you please confirm you don't use that `static final String CURRENT_PROD_VERSION = '7.12.2'` anymore?

Reply:
IIRC this env is not used in the nightly images pipeline. We do use the NEXT_PROD_VERSION and BAMOE_NEXT_PROD_VERSION.

Reply:
@spolti so we can remove both BAMOE_CURRENT_PROD_VERSION and CURRENT_PROD_VERSION variables?

Reply:
I think so.